packages/foundations-forms/src/forms.pcss

```diff
@@ -145,20 +145,44 @@ input,
   }
 }
 
+/*
+  Fixed width text inputs
+  1x Avenir Next "W" character = 2.5ch
+
+  Support: IE9+
+  https://developer.mozilla.org/en-US/docs/Web/CSS/length
+*/
+
+.coop-form__input--width-1 {
+  max-width: calc((1 * 2ch) + 1.5rem);
```

Comment:
Could `2ch` and `1.5rem` be moved to vars?

Reply:
@gbbns Yeah, definitely. It means we have to publish every single package 🙈 

I'm up for it if you are?